src/components/Search/SearchResultsListItem.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import Checkbox from '@mui/material/Checkbox';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import PropTypes from 'prop-types';
+
+const truncateHash = (item) => item.revision.slice(0, 11);
+
+const getLastRevision = (item) => item.revisions[item.revisions.length - 1];
+
+function SearchResultsListItem(props) {
+  const { item } = props;
+  return (
+    <ListItemButton key={item.id}>
+      <ListItem className="search-revision-item search-revision" disablePadding>
+        <ListItemIcon className="search-revision-item-icon search-revision">
+          <Checkbox
+            className="search-revision-item-checkbox search-revision"
+            edge="start"
+            tabIndex={-1}
+            disableRipple
+          />
```

Comment:
added checkboxes to select multiple revisions. Will need to add validation in the future to prevent selecting more than four.

Reply:
I like it!